### PR TITLE
- updating repo to face RN 0.2.6

### PR DESCRIPTION
**Pull request by SencyCaspit**

### CODE REVIEW

File: ios/Podfile.lock [hunk 1]
```diff
@@ -1231,7 +1231,7 @@ DEPENDENCIES:
   - React-jsitracing (from `../node_modules/react-native/ReactCommon/hermes/executor/`)
   - React-logger (from `../node_modules/react-native/ReactCommon/logger`)
   - React-Mapbuffer (from `../node_modules/react-native/ReactCommon`)
-  - "react-native-smkit-ui (from `../node_modules/@sency/react-native-smkit-ui`)"
+  - "react-native-smkit-ui (from `../node_modules/@sency/react-native-smkit-ui-dev`)"
```
Comment (ofergoldstein) [inline ios/Podfile.lock:1234]: @SencyCaspit dev?

File: ios/Podfile.lock [hunk 2]
```diff
@@ -1330,7 +1330,7 @@ EXTERNAL SOURCES:
   React-Mapbuffer:
     :path: "../node_modules/react-native/ReactCommon"
   react-native-smkit-ui:
-    :path: "../node_modules/@sency/react-native-smkit-ui"
+    :path: "../node_modules/@sency/react-native-smkit-ui-dev"
```
Comment (ofergoldstein) [inline ios/Podfile.lock:1333]: @SencyCaspit dev?

File: ios/Podfile.lock [hunk 3]
```diff
@@ -935,7 +935,7 @@ PODS:
   - React-Mapbuffer (0.74.2):
     - glog
     - React-debug
-  - react-native-smkit-ui (0.2.5):
+  - react-native-smkit-ui (0.2.6-snapshot1):
```
[the remark this answers is not in the document]
Comment (SencyCaspit) [reply ios/Podfile.lock:938]: @ofergoldstein those are in the Podfile.lock, i uploaded the rn version to our private repo in npm, to test the plugin. its not included in the project

File: ios/Podfile.lock (hunk 1 above)
Comment (SencyCaspit) [reply ios/Podfile.lock:1234]: @ofergoldstein same

File: ios/Podfile.lock (hunk 2 above)
Comment (SencyCaspit) [reply ios/Podfile.lock:1333]: @ofergoldstein same

**Review by ofergoldstein — APPROVED** [review]

**Pull request merged**